### PR TITLE
crossing-training.lic: Don't train stealth above 30 ranks.

### PR DESCRIPTION
**Pull request by Sarvatt**
It is just a waste of time to train this higher, and is really obvious behavior. People think it actually trains when they first join lich without reading the scripting reference saying its low ranks only leading to spam at the empaths guild.

### CODE REVIEW

File: crossing-training.lic [hunk 1]
```diff
@@ -738,11 +738,17 @@ class CrossingTraining
...
   def train_stealth
-    hide_in 851
-    hide_in 850
-    hide_in 764
-    hide_in 5992
-    hide_in 992
+    if DRSkill.getrank('Stealth') >= 25
+      echo('Skill too high to learn this way, removing Stealth from training')
+      @settings.crossing_training.delete('Stealth')
+      nil
```
Comment (Sarvatt) [inline crossing-training.lic]: hmm this was return but rubocop changed it to nil

Comment (jandersson) [reply crossing-training.lic]: Since ruby returns the last thing evaluated, `return` and `nil` would be the same

File: crossing-training.lic [hunk 2]
```diff
@@ -738,11 +738,16 @@ class CrossingTraining
...
 
   def train_stealth
-    hide_in 851
-    hide_in 850
-    hide_in 764
-    hide_in 5992
-    hide_in 992
+    if DRSkill.getrank('Stealth') >= 30
+      echo('Skill too high to learn this way, removing Stealth from training')
+      @settings.crossing_training.delete('Stealth')
```
Comment (rpherbig) [inline crossing-training.lic]: I prefer something like
```
if ...
    ...
    return
end

hide_in ...
```

It's a completely stylistic thing, so don't worry about changing it, I just wanted to point out an alternative.

Comment (Sarvatt) [reply crossing-training.lic]: that's what I had originally, I'll use that style in the future.

**Pull request merged**